bin/sqs_cli.py

```diff
@@ -10,41 +10,21 @@
 #
 # Usage: socorro-cmd sqs [SUBCOMMAND]
 
-import os
 import sys
 import time
 
-import boto3
 import click
 
-
-VISIBILITY_TIMEOUT = 2
-
-
-class InvalidQueueName(Exception):
-    """Denotes an invalid queue name."""
+from socorro import settings
+from socorro.libclass import import_class
 
 
-def validate_queue_name(queue_name):
-    if len(queue_name) > 80:
-        raise InvalidQueueName("queue name is too long.")
-
-    for c in queue_name:
-        if not c.isalnum() and c not in "-_":
-            raise InvalidQueueName("%r is not an alphanumeric, - or _ character." % c)
+VISIBILITY_TIMEOUT = 2
 
 
 def get_client():
-    session = boto3.session.Session(
-        aws_access_key_id=os.environ.get("resource.boto.access_key"),
-        aws_secret_access_key=os.environ.get("secrets.boto.secret_access_key"),
-    )
-    client = session.client(
-        service_name="sqs",
-        region_name=os.environ.get("resource.boto.region"),
-        endpoint_url=os.environ.get("resource.boto.sqs_endpoint_url"),
-    )
-    return client
+    cls = import_class(settings.QUEUE["class"])
+    return cls.build_client(**settings.QUEUE["options"])
```

Comment:
This is weird, but what's going on is that we can't instantiate a `SQSCrashQueue` because it tries to get the queue urls for the standard, priority, and reprocessing queues and if those queues haven't been created, yet, then it fails. Given that this is the script that the local dev environment uses to create, delete, and get information from the state of SQS in the local dev environment, we can't use `SQSCrashQueue` and have to use the boto client directly.